unit/libcprover-cpp/CMakeLists.txt

```diff
@@ -5,9 +5,11 @@ add_executable(lib-unit ${sources})
 target_include_directories(lib-unit
     PUBLIC
     ${CBMC_SOURCE_DIR}/libcprover-cpp
+    ${CBMC_SOURCE_DIR}/unit/testing-utils
```

Comment:
See if it is to be removed - we don't want to expose symbols from the `libcproverunit` binary.